scripts/upgradeable/transpile.sh

```diff
@@ -29,7 +29,7 @@ npx @openzeppelin/upgrade-safe-transpiler@latest -D \
   -x 'contracts/proxy/**/*' \
   -x '!contracts/proxy/Clones.sol' \
   -x '!contracts/proxy/ERC1967/ERC1967Storage.sol' \
-  -x '!contracts/proxy/ERC1967/ERC1967Upgrade.sol' \
+  -x '!contracts/proxy/ERC1967/ERC1967Utils.sol' \
```

Comment:
Aren't we ignoring libraries for transpilation already?

If so, let's also create an issue with good-first-issue label

Reply:
Basically the library are needed by the contracts, so if we don't transpile them we need to get them from somewhere.

The partial transpilation proposal is to not transpile the libraries and the interfaces, and import the vanila version instead. (the creates a dependency from the upgradeable contracts to the vanilla contracts)